content/en/docs/ambient/usage/ambient-and-networkpolicy/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Ambient and Kubernetes NetworkPolicy
```

Comment:
Is it too vague to go with "network policies" in text?

Reply:
It's a specific K8S API and not network policies as a concept, so I think we have to use the proper name.